vhdbuilder/packer/install-dependencies.sh

```diff
@@ -561,10 +561,14 @@ fi
 # Please do not use the .1 suffix, because that's only for the base image patches
 # regular version >= v1.17.0 or hotfixes >= 20211009 has arm64 binaries.
 KUBE_BINARY_VERSIONS="$(jq -r .kubernetes.versions[] manifest.json)"
+KUBE_URL="$(jq -r .kubernetes.downloadURL manifest.json)"
 
 for PATCHED_KUBE_BINARY_VERSION in ${KUBE_BINARY_VERSIONS}; do
   KUBERNETES_VERSION=$(echo ${PATCHED_KUBE_BINARY_VERSION} | cut -d"_" -f1 | cut -d"-" -f1 | cut -d"." -f1,2,3)
-  extractKubeBinaries $KUBERNETES_VERSION "https://acs-mirror.azureedge.net/kubernetes/v${PATCHED_KUBE_BINARY_VERSION}/binaries/kubernetes-node-linux-${CPU_ARCH}.tar.gz" false
+  KUBE_VERSION_URL="${KUBE_URL/\$\{PATCHED_KUBE_BINARY_VERSION\}/${PATCHED_KUBE_BINARY_VERSION}}"
+  KUBE_VERSION_URL="${KUBE_VERSION_URL/\${CPU_ARCH\}/${CPU_ARCH}}"
```

Comment:
any reason why this cannot be combined with the previous line ?    when reading the code, I was wondering why you were setting the same VAR twice, till I realized you were augmenting it.

Reply:
It took me a bit to figure out the right shell syntax. I've modified it now to use "envsubst" as that seems much cleaner.